src/http/api/feed.ts

```diff
@@ -71,7 +71,9 @@ export function createGetFeedHandler(
                 likedByMe: result.post_liked_by_user === 1,
                 replyCount: result.post_reply_count,
                 readingTimeMinutes: result.post_reading_time_minutes,
-                attachments: [], // TODO
+                attachments: result.post_attachments
+                    ? result.post_attachments
+                    : [],
```

Comment:
Rather than trusting that whats in the field is correct, i think we should explicitly map out the values so we can guarantee we are returning the correct thing in the correct structure